tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -87,6 +88,11 @@ public static void main(String[] args) throws Exception {
                     payloadByteList.add(payload.getBytes(StandardCharsets.UTF_8));
                 }
             }
+            
+            Properties headerProps = new Properties();
+            if (headersPath != null) {
+                headerProps.putAll(Utils.loadProps(headersPath));
```

Comment:
@vgvineet4 Thanks for the PR. we are adding same header values for each record. Should we set different values for each record?  maybe add string header value?

Reply:
@omkreddy Thanks for reviewing the PR. Line number 142 <code>payload = payloadByteList.get(random.nextInt(payloadByteList.size()));</code> pulls the record in file randomly. It would be hard to map headers to the required payload. My thinking was this could serve well for those who want to check against a particular workflow where the payload or data in each record varies but header remain constant.